Tests/RegressionTests.cs

```diff
@@ -150,21 +150,21 @@ private static TestCaseData[] GetRegressionTestParameters()
                 {"Total Trades", "1"},
                 {"Average Win", "0%"},
                 {"Average Loss", "0%"},
-                {"Compounding Annual Return", "155.210%"},
+                {"Compounding Annual Return", "191.086%"},
```

Comment:
Lets aim for getting this working without regression changes first to be safe. Its a fairly extensive change :) Test code might need updating to ensure they give rounded quantities.